src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,61 @@
+package mate.academy.model;
+
+import java.util.Arrays;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;

```

Comment:
"Let’s make the login unique. "
// I think, it's about email, because we can't have two equal email